src/test/java/org/takes/facets/hamcrest/HmRqBodyTest.java

```diff
@@ -67,5 +71,28 @@ public void testsBodyValuesAreDifferent() {
             Matchers.not(new HmRqBody("that"))
         );
     }
-}
 
+    /**
+     * HmRqBody can describe mismatch in descriptive way.
+     */
+    @Test
+    public void mismatchMessageIsReadable() {
```

Comment:
@t-izbassar It's better to call test methods as described in "Test Method Name" section in http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html to keep code more consistent